src/bp/core/config/config-loader.ts

```diff
@@ -129,18 +129,20 @@ export class ConfigProvider {
   }
 
   public async getModulesListConfig() {
-    const enabledByDefault = [
-      'analytics',
-      'basic-skills',
-      'builtin',
-      'channel-web',
-      'nlu',
-      'qna',
-      'extensions',
-      'code-editor',
-      'testing',
-      'examples'
-    ]
+    const enabledByDefault: string[] = process.env.TEST_NLU
+      ? ['nlu', 'nlu-testing']
+      : [
+          'analytics',
+          'basic-skills',
+          'builtin',
+          'channel-web',
+          'nlu',
+          'qna',
+          'extensions',
+          'code-editor',
+          'testing',
+          'examples'
+        ]
```

Comment:
When using environment variables, please use yn(), because setting `process.env.TEST_NLU = false` or `process.env.TEST_NLU = true` would give the same result